src/js/util/thumbnails.js

```diff
@@ -19,6 +19,7 @@ const endpoints = {
   pixiv: 'http://embed.pixiv.net/embed_json.php?callback=callback&size=medium&id=',
   tinami: 'https://www.tinami.com/api/content/info?',
   nicoseiga: 'http://ext.seiga.nicovideo.jp/thumb/',
+  googleplus: 'https://picasaweb.google.com/data/feed/api/',
 };
```

Comment:
Could you add this endpoint to `tools/manifests/common.js` in the `permissions` field as well?